src/targets/gpu/device/include/migraphx/gpu/device/reduce.hpp

```diff
@@ -35,7 +35,12 @@ inline namespace MIGRAPHX_INLINE_NS {
 namespace gpu {
 namespace device {
 
+#ifdef MIGRAHPX_USE_HIPRTC
+#define MIGRAPHX_NO_DPP
+#endif
```

Comment:
These changes should be removed. This file is not ever compiled with hiprtc.

Reply:
Done.